src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+public class Ui {

```

Comment:
I like how you organise the print system together to allows for easier modification.